src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+}
+
+.stars--1 :nth-child(-n + 1),
```

Comment:
It will be better if u make like this, and put it into .stars block
```
&--1 :nth-child(-n + 1),
&--2 :nth-child(-n + 2),
&--3 :nth-child(-n + 3),
&--4 :nth-child(-n + 4),
&--5 :nth-child(-n + 5) {
  background-image: url(../images/star-active.svg);` 
```